### PR TITLE
Zoom controls in toolbar

### PR DESCRIPTION
**Pull request by Gnafu**
Created ZoomIn and ZoomOut plugins

### CODE REVIEW

File: web/client/components/buttons/ZoomButton.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    getDefaultProps() {
+        return {
+            id: "mapstore-zoom",
+            className: "square-button",
+            glyphicon: "plus",
+            btnSize: 'xsmall',
+            step: 1,
+            currentZoom: 3,
+            minZoom: 0,
+            maxZoom: 40,
```
Comment (mbarto) [inline web/client/components/buttons/ZoomButton.jsx]: Let's use a more common value (leaflet / ol3 default max zoom ?)

File: web/client/plugins/ZoomIn.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const assign = require('object-assign');
+
+module.exports = {
+    ZoomInPlugin: assign(ZoomInButton, {
+        Toolbar: {
+            name: "ZoomIn",
+            position: 1,
+            tooltip: "zoombuttons.zoomInTooltip",
+            tool: true,
+            hide: true
```
Comment (mbarto) [inline web/client/plugins/ZoomIn.jsx]: add glyphicon config here

File: web/client/plugins/ZoomOut.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+const assign = require('object-assign');
+
+module.exports = {
+    ZoomOutPlugin: assign(ZoomOutButton, {
+        Toolbar: {
+            name: "ZoomOut",
+            position: 2,
+            tool: true,
+            tooltip: "zoombuttons.zoomOutTooltip",
+            hide: true
```
Comment (mbarto) [inline web/client/plugins/ZoomOut.jsx]: add glyphicon config here

File: web/client/plugins/ZoomIn.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+ /**
+ * Plugin for Zoom in
+ */
+const {connect} = require('react-redux');
+const {createSelector} = require('reselect');
+const {mapSelector} = require('../selectors/map');
+
+const selector = createSelector([mapSelector], (map) => ({currentZoom: map && map.zoom, id: "zoomin-btn"}));
```
Comment (mbarto) [inline web/client/plugins/ZoomIn.jsx]: configure step here

File: web/client/plugins/ZoomOut.jsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+ /**
+ * Plugin for Zoom out
+ */
+const {connect} = require('react-redux');
+const {createSelector} = require('reselect');
+const {mapSelector} = require('../selectors/map');
+
+const selector = createSelector([mapSelector], (map) => ({currentZoom: map && map.zoom, id: "zoomin-btn"}));
```
Comment (mbarto) [inline web/client/plugins/ZoomOut.jsx]: configure step here

File: web/client/localConfig.json [hunk 6]
```diff
@@ -114,6 +119,22 @@
...
+      "hide": true,
+      "cfg":{
+        "step": 1,
+        "glyphicon": "plus"
+        }
+    },
+    {
+      "name":"ZoomOut",
+      "hide": true,
+      "cfg":{
```
Comment (mbarto) [inline web/client/localConfig.json]: no cfg should be needed for default behaviour

File: web/client/localConfig.json [hunk 7]
```diff
@@ -114,6 +119,22 @@
       "name": "ZoomAll",
       "hide": true
     },
-    "MapLoading", "Snapshot"]
+    "MapLoading", "Snapshot",
+    {
+      "name":"ZoomIn",
+      "hide": true,
+      "cfg":{
```
Comment (mbarto) [inline web/client/localConfig.json]: no cfg should be needed for default behaviour

File: web/client/plugins/ZoomIn.jsx (hunk 2 above)
Comment (Gnafu) [inline web/client/plugins/ZoomIn.jsx]: I'm using tool:true, the button is not rendered by the toolbar

**Comment by Gnafu** [comment]
Updated, waiting for turf repo to be fixed, so travis can build

**Pull request merged**